Samples/outlook-spam-reporting/README.md

```diff
@@ -39,59 +39,145 @@ To learn about key components of this sample, see [Implement an integrated spam-
+    npm stop
     ```
 
-    This command will display the folder location where it generated the certificate files.
+#### Use Microsoft Azure
+
+You can deploy this sample with the unified manifest to Microsoft Azure using the Teams Toolkit extension in Visual Studio Code.
+
+1. In Visual Studio Code, go to the activity bar, then open the Teams Toolkit extension.
+1. In the Accounts section of the Teams Toolkit pane, choose **Sign in to Azure**.
+1. After you sign in, select a subscription under your account.
+1. In the Development section of the Teams Toolkit pane, choose **Provision in the cloud**. Alternatively, open the command palette and choose **Teams: Provision in the cloud**.
+1. Choose **Deploy to the cloud**. Alternatively, open the command palette and choose **Teams: Deploy to the cloud**.
 
-1. Go to the folder location where the certificate files were generated, then copy the **localhost.crt** and **localhost.key** files to the cloned or downloaded sample folder.
+Once the sample is successfully deployed, follow these steps.
+
+1. Copy the endpoint of your new Azure deployment. Use one of the following methods.
+    - In Visual Studio Code, select **View** > **Output** to open the Output window. Then, copy the endpoint for your new Azure deployment.
+    - In the Azure portal, go to the new storage account. Then, choose **Data management** > **Static website** and copy the **Primary endpoint** value.
+1. Open the **./webpack.config.js** file.
+1. Change the `urlProd` constant to use the endpoint of your Azure deployment.
+1. Save your change. Then, run the following command.
+
+    ```console
+    npm run build
+    ```
+
+    This generates a new **manifest.json** file in the **dist** folder of your project that will load the add-in resources from your storage account.
 1. Run the following command.
 
     ```console
-    http-server -S -C localhost.crt -K localhost.key --cors . -p 3000
+    npm run start:prod
+    ```
+
+    Classic Outlook on Windows starts and the **manifest.json** file is sideloaded from the **dist** folder.
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+1. To stop the web server and uninstall the add-in from Outlook, run the following command.
+
+    ```console
+    npm run stop:prod
+    ```
+
+### Run with the add-in only manifest
+
+#### Use GitHub as the web host
+
+The quickest way to run the sample is to use GitHub as the web host. However, you can't debug or change the source code. The add-in web files are served from this GitHub repository.
+
+1. Download the **manifest.xml** file from this sample to a folder on your computer.
+1. Sideload the add-in only manifest in Outlook on the web or on Windows (new or classic) by following the manual instructions in [Sideload Outlook add-ins for testing](https://learn.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing#sideload-manually).
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+
```

Comment:
Should we tell readers how to stop and uninstall?

Reply:
Added.